crates/solver/src/solver/baseline_solver.rs

```diff
@@ -829,7 +829,7 @@ mod tests {
         );
         assert_eq!(
             pool_1.get_amount_in(tokens[0], (1.into(), tokens[1])),
-            Some(8999613.into())
+            Some(15999226.into())
```

Comment:
Is this change (and later ones) due to the change in the convergence function?

Reply:
Yes.